source/server/BUILD

```diff
@@ -70,16 +71,42 @@ envoy_cc_library(
     name = "connection_handler_impl",
     srcs = ["connection_handler_impl.cc"],
     hdrs = [
+        "active_tcp_listener.h",
```

Comment:
This shouldn't be here. It should get picked up via a dep on `active_tcp_listener`.